{{cookiecutter.repo_name}}/.github/workflows/CI.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
     - name: CodeCov
       uses: codecov/codecov-action@v1
       with:
-        token: ${{ secrets.CODECOV_TOKEN }}
+        token: {{ '${{ secrets.CODECOV_TOKEN }}' }}
```

Comment:
I don't think the braces are needed here? This isn't needed on my yaml.

Reply:
We need to escape them so `cookiecutter`'s Jinja processing doesn't parse it. It's also needed on `runs-on`, for example.

Reply:
Ah right, its rendered...